### DISCUSSION TITLE
Enforcing Company Email Registration in GitHub Profiles

### DISCUSSION THREAD

**Discussion by ot-net** [community/community#75468] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I am seeking a method to require users to register their company email addresses in their GitHub profiles. The objective is to ensure that commits are made using company emails, forcing users to add their business email to their GitHub accounts, in order to be part of an organization...

Is there a way to implement domain restrictions specifically for members within an organization? Any insights or guidance on achieving this would be greatly appreciated.

**Comment by shazib-summar** [comment]
@ge-infosec were you able to find anything on this?
